dockerfiles/che/launcher.sh

```diff
@@ -0,0 +1,373 @@
+#!/bin/sh
+
+init_logging() {
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  RED='\033[0;31m'
+  NC='\033[0m'
+}
+
+init_global_variables() {
+
+  CHE_SERVER_CONTAINER_NAME="che-server"
+  CHE_SERVER_IMAGE_NAME="codenvy/che"
+  CHE_LAUNCER_IMAGE_NAME="codenvy/che-launcher"
+
+  # Possible Docker install types are:
+  #     native, boot2docker or moby
+  DOCKER_INSTALL_TYPE=$(get_docker_install_type)
+
+  # User configurable variables
+  DEFAULT_DOCKER_HOST_IP=$(get_docker_host_ip)
+  DEFAULT_CHE_HOSTNAME=$(get_che_hostname)
+  DEFAULT_CHE_PORT="8080"
+  DEFAULT_CHE_VERSION="latest"
+  DEFAULT_CHE_RESTART_POLICY="always"
+  DEFAULT_CHE_USER="root"
+  DEFAULT_CHE_LOG_LEVEL="info"
+  DEFAULT_CHE_DATA_FOLDER="/home/user/che"
+
+  CHE_HOSTNAME=${CHE_HOSTNAME:-${DEFAULT_CHE_HOSTNAME}}
+  CHE_PORT=${CHE_PORT:-${DEFAULT_CHE_PORT}}
+  CHE_VERSION=${CHE_VERSION:-${DEFAULT_CHE_VERSION}}
+  CHE_RESTART_POLICY=${CHE_RESTART_POLICY:-${DEFAULT_CHE_RESTART_POLICY}}
+  CHE_USER=${CHE_USER:-${DEFAULT_CHE_USER}}
+  CHE_HOST_IP=${CHE_HOST_IP:-${DEFAULT_DOCKER_HOST_IP}}
+  CHE_LOG_LEVEL=${CHE_LOG_LEVEL:-${DEFAULT_CHE_LOG_LEVEL}}
+  CHE_DATA_FOLDER=${CHE_DATA_FOLDER:-${DEFAULT_CHE_DATA_FOLDER}}
+
+  # CHE_CONF_ARGS are the Docker run options that need to be used if users set CHE_CONF_FOLDER:
+  #   - empty if CHE_CONF_FOLDER is not set
+  #   - -v ${CHE_CONF_FOLDER}:/conf -e "CHE_LOCAL_CONF_DIR=/conf" if CHE_CONF_FOLDER is set
+  CHE_CONF_ARGS=${CHE_CONF_FOLDER:+-v ${CHE_CONF_FOLDER}:/conf -e \"CHE_LOCAL_CONF_DIR=/conf\"}
+  CHE_LOCAL_BINARY_ARGS=${CHE_LOCAL_BINARY:+-v ${CHE_LOCAL_BINARY}:/home/user/che}
+
+  USAGE="
+Usage:
+  docker run -v /var/run/docker.sock:/var/run/docker.sock ${CHE_LAUNCER_IMAGE_NAME} [COMMAND]
```

Comment:
I've updated `launcher.sh`. Now Che is started in debug mode if the env variable `CHE_LOG_LEVEL` value is `debug`.